generators/java/sdk/src/main/java/com/fern/java/client/generators/endpoint/AbstractEndpointWriter.java

```diff
@@ -346,7 +349,7 @@ protected final MethodSpec getEnvironmentToUrlMethod() {
     }
 
     public final String getVariableName(String variable) {
-        if (this.endpointParameterNames.contains("body")) {
+        if (this.endpointParameterNames.contains(variable)) {
```

Comment:
this was a small separate fix -- method was only being used with a `variable` of `body` so there were no bugs, but could led have led to future oens